### DISCUSSION TITLE
42 and new

### DISCUSSION THREAD

**Discussion by phoenixonfire9180** [community/community#43658] [New to GitHub]
I'm new here and wondering how best to get started.

**Comment by jongwooo** [comment]
@phoenixonfire9180 This [document](https://docs.github.com/en/get-started/quickstart/hello-world) may be helpful.

**Comment by queenofcorgis** [comment]
Welcome to GitHub and the GitHub Community @phoenixonfire9180 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist. 

Best of luck!
